### PR TITLE
fixed spelling mistakes.

### PR DESCRIPTION
**Pull request by ghuntley**
happy #dockerbday

Signed-off-by: Geoffrey Huntley ghuntley@ghuntley.com

### CODE REVIEW

File: vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Like the previous command, you can run this on OSX too!
 
 ### Building libcontainer directly
 
-> Note: You should add the `vendor` directory to your GOPATH to use the vendored libraries
+> Note: You should add the `vendor` directory to your GOPATH to use the vendor libraries
```
Comment (unclejack) [inline vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md]: `vendored` is also used in other contexts, like when discussing ruby gems and other things.

Comment (ghuntley) [inline vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md]: I can change this back to ```vendored`` if need be?

Comment (SvenDowideit) [inline vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md]: vendored is correct in this context - meaning adding a particular version of an external library to your project's revision control system.

**Comment by moxiegirl** [comment]
LGTM

**Comment by SvenDowideit** [comment]
LGTM - I think in this context, the use of vendor is reasonable and correct.

**Comment by crosbymichael** [comment]
Thanks but this change has to be made in https://github.com/docker/libcontainer as you cannot changed vendored dependencies outside of the main repository.  

Would you mind moving your change over to the libcontainer repo?

Thanks again!!

**Comment by ghuntley** [comment]
@crosbymichael is was a change to the *documentation*.

The debate is above about wether "vendored" is even a word.

File: vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md (hunk 1 above)
Comment (ghuntley) [inline vendor/src/github.com/docker/libcontainer/CONTRIBUTING.md]: I  can amend back to ```vendored``` once @crosbymichael or @SvenDowideit reopen the pull request. Once done we can LGTM/merge.

**Pull request closed without merging**